pkg/cluster/loadbalancerprofile.go

```diff
@@ -224,29 +269,38 @@ func (m *manager) getDesiredOutboundIPs(ctx context.Context) ([]api.ResourceRefe
 	return nil, nil
 }
 
+type createIPResult struct {
```

Comment:
Do we have any convention around grouping type definitions at the top of a file or do we just define them before they are used like this as the norm?

Reply:
I agree that these should be at the top, I'll move them.